semgrep.yml

```diff
@@ -111,6 +111,10 @@ rules:
       - pattern: Common.map ... |> List.flatten
       - pattern: List.map ... |> List.concat
       - pattern: Common.map ... |> List.concat
+      - pattern: ... |> List.map ... |> List.flatten
+      - pattern: ... |> Common.map ... |> List.flatten
+      - pattern: ... |> List.map ... |> List.concat
+      - pattern: ... |> Common.map ... |> List.concat
```

Comment:
Not sure it this will work since OCaml isn't GA but in principle one could reduce the combinatorial explosion by matching `$MAP` and `$CONCAT` and then metavaraible-pattern each of these variables, e.g.:
```
metavaraible-pattern:
  metavariable: $CONCAT
  pattern-either:
  - pattern: List.flatten
  - pattern: List.concat
```

Reply:
I hope this is possible. I'll try it next time.